documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -9,10 +9,12 @@ description: "How to install, upgrade, and uninstall the operator."
 
 - [Introduction](#introduction)
 - [Install the operator](#install-the-operator)
+- [Install the WebLogic domain resource conversion webhook](#install-the-weblogic-domain-resource-conversion-webhook)
 - [Set up domain namespaces](#set-up-domain-namespaces)
 - [Update a running operator](#update-a-running-operator)
 - [Upgrade the operator](#upgrade-the-operator)
 - [Uninstall the operator](#uninstall-the-operator)
+- [Uninstall the WebLogic domain resource conversion webhook](#uninstall-the-weblogic-domain-resource-conversion-webhook)
```

Comment:
There is no corresponding section in the doc called "Uninstall the WebLogic domain resource conversion webhook"

Reply:
Fixed the TOC, this section was removed as per the earlier PR review comments.